tests/package_tests/package_test.py

```diff
@@ -0,0 +1,220 @@
+        subprocess.check_call(
+            f"scalyr-agent-2 status -v", shell=True
+        )
+    elif package_type == "tar":
+        tarball_dir = list(pl.Path("~").expanduser().glob("scalyr-agent-*.*.*"))[0]
+
+        binary_path = tarball_dir / "bin/scalyr-agent-2"
+        subprocess.check_call(
+            [binary_path, "status", "-v"]
+        )
+
+
+def stop_agent():
+    if package_type in ["deb", "rpm", "msi"]:
+        subprocess.check_call(
+            f"scalyr-agent-2 stop", shell=True
+        )
+    if package_type == "tar":
+        tarball_dir = list(pl.Path("~").expanduser().glob("scalyr-agent-*.*.*"))[0]
+
+        binary_path = tarball_dir / "bin/scalyr-agent-2"
+        subprocess.check_call(
+            [binary_path, "stop"]
+        )
+
+
+def configure_agent(api_key: str):
+    if package_type in ["deb", "rpm"]:
+        config_path = pathlib.Path(AGENT_CONFIG_PATH)
+    elif package_type == "tar":
+        version = determine_version()
+
+        config_path = pl.Path("~").expanduser() / f"scalyr-agent-{version}" / "config" / "agent.json"
+    elif package_type == "msi":
+        config_path = pl.Path(os.environ["programfiles(x86)"], "Scalyr", "config", "agent.json")
+
+    config = {}
+    config["api_key"] = api_key
+
+    config["server_attributes"] = {"serverHost": "ARTHUR_TEST"}
+    config["implicit_metric_monitor"] = False
+    config["verify_server_certificate"] = False
+    config_path.write_text(json.dumps(config))
+
+
+def remove_deb_package():
+    subprocess.check_call(
+        f"apt-get remove -y scalyr-agent-2",
+        shell=True
+    )
+
+
+def remove_rpm_package():
+    subprocess.check_call(
+        f"yum remove -y scalyr-agent-2",
+        shell=True
+    )
+
+
+def remove_package(package_type: str):
+    if package_type == "deb":
+        remove_deb_package()
+    elif package_type == "rpm":
+        remove_rpm_package()
+
+
+AGENT_CONFIG_PATH ="/etc/scalyr-agent-2/agent.json"
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--package-path",
+        type=str,
+        required=True
+    )
+
+    parser.add_argument("--scalyr-api-key", required=True)
+
+    args = parser.parse_args()
+
+    package_path = pl.Path(args.package_path)
+
+    scalyr_api_key = args.scalyr_api_key
+
+    if not package_path.exists():
+        print("No package.")
+        exit(1)
+
+    if package_path.name.endswith(".deb"):
+        package_type = "deb"
+    elif package_path.name.endswith(".rpm"):
+        package_type = "rpm"
+    elif package_path.name.endswith("tar.gz"):
+        package_type = "tar"
+    elif package_path.name.endswith(".msi"):
+        package_type = "msi"
+    else:
+        raise ValueError("Unknown package format.")
+
+    install_package(package_type)
+
+    configure_agent(args.scalyr_api_key)
+
+    start_agent()
+
+    time.sleep(2)
+
+    get_agent_status()
+
+    time.sleep(2)
+
+    stop_agent()
```

Comment:
Do we want to verify that stop command indeed works correctly as expected?

Reply:
Same as in that comment. https://github.com/scalyr/scalyr-agent-2/pull/764#discussion_r682772312